src/components/Disputes/DisputeDetail.js

```diff
@@ -1,35 +1,53 @@
-import React from 'react'
-import { BackButton, Bar, Box, Split } from '@aragon/ui'
-
+import React, { useCallback, useMemo } from 'react'
+import { Header, SyncIndicator, BackButton, Bar, Box, Split } from '@aragon/ui'
+import { useHistory } from 'react-router-dom'
+import useDisputesSubscription from './hooks/useDisputesSubscription'
 import DisputeInfo from './DisputeInfo'
 import DisputeEvidences from './DisputeEvidences'
 import DisputeTimeline from './DisputeTimeline'
 
-function DisputeDetail({ dispute, onBack }) {
+function DisputeDetail({ match }) {
+  const history = useHistory()
+  const disputes = useDisputesSubscription()
```

Comment:
Should we do a dispute subscription by the id  instead of subscribing to all disputes?

Reply:
Done: https://github.com/aragon/court-dashboard/pull/41/commits/c6e04591323e75c22cdbc55aff4acfc5d2aa79d5 and https://github.com/aragon/court-dashboard/pull/41/commits/4343c283e43e8bf5f14c5dfde07ee80446cf5357